recipes/dpdispatcher/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dpdispatcher" %}
+{% set version = "0.4.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0e177258fbf7ac6ce595933d669366723bd8f851448df2216b3efdb7b5434952
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - dpdisp = dpdispatcher.dpdisp:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
+    - dargs
```

Comment:
It looks like the `host` section probably only needs `python`, `pip` and `setuptools_scm`. I think the rest are just listed under `install_requires`, not `setup_requires`.

Reply:
Done.